shared/src/main/scala/Elements.scala

```diff
@@ -482,9 +486,10 @@ object Elements:
       "a: str, b: num|str -> does regex pattern b match haystack a?"
     ) {
       case (a: VNum, b: VNum) =>
-        NumberHelpers.range(a, b - 1)
-      case (a: String, b: String) => a.r.findFirstIn(b).isDefined
-      case (a: String, b: VNum)   => a.r.findFirstIn(b.toString).isDefined
+        NumberHelpers.range(a, b).dropRight(1)
```

Comment:
May I ask why dropRight is needed instead of the b-1 from before?

Reply:
Because of cases like `5 -> 1`

Reply:
A range from 5 to 0 isn't what should be returned. So instead of having conditional swapping, just remove the last item of the range. That works for ranges in both directions.

Reply:
Oh, smart